lib/seattleflu/db/cli/command/etl/presence_absence.py

```diff
@@ -0,0 +1,386 @@
+"""
+Process enrollment documents into the relational warehouse
+"""
+import click
+import logging
+import sys
```

Comment:
This import is unused, as is the import of `itemgetter` below. Would you double check the rest and remove what's not used?